conf/defaults.ini

```diff
@@ -866,6 +866,11 @@ max_annotations_to_keep =
 # Enable the Explore section
 enabled = true
 
+#################################### Query History #############################
+[query_history]
+# Enable the Query history
+enabled = false
```

Comment:
Currently set to false, you have to enable it for tests. When the feature is released, we can change the default to true.